packages/asset-swapper/test/market_operation_utils_test.ts

```diff
@@ -759,6 +867,62 @@ describe('MarketOperationUtils tests', () => {
                 ];
                 expect(orderSources).to.deep.eq(expectedSources);
             });
+
+            it('is able to create a order from LiquidityProvider', async () => {
+                const registryAddress = randomAddress();
+                const liquidityProviderAddress = randomAddress();
+                const xAsset = randomAddress();
+                const yAsset = randomAddress();
+                const toSell = Web3Wrapper.toBaseUnitAmount(10, 18);
```

Comment:
Nit: There's a better version of this in `@0x/utils` (`fromTokenUnitAmount()`, `toTokenUnitAmount()`).

Reply:
Thanks! I didn't know about this utility :)

Reply:
fixed